specs/commandline/policy.md

```diff
@@ -0,0 +1,232 @@
+                "expiry" : "log"
+              }
+            },
+            "trustStores": ["ca:acme-rockets"],
+            "trustedIdentities": ["*"]
+        },
+        {
+            "name": "global-policy-for-all-other-images",
+            "registryScopes": [ "*" ],       
+            "signatureVerification": {                                
+                "level": "strict"
+            },
+            "trustStores": [ 
+              "ca:acme-rockets"
+            ],                  
+            "trustedIdentities": [                                    
+                "x509.subject: C=US, ST=WA, L=Seattle, O=acme-rockets.io, CN=SecureBuilder"
+            ]
+        }
+    ]
+}
+```
+
+## Outline
+
+### notation policy command
+
+```text
+Manage trust policies for signature verification.
+
+Usage:
+  notation policy [command]
+
+Available Commands:
+  import    import trust policies from a JSON file
+  init      initialize trust policies with default values
+  show      show trust policies
+
+Flags:
+  -h, --help   help for policy
+```
+
+### notation policy import
+
+```text
+Import trust policies from a JSON file
+
+Usage:
+  notation policy import [flags] <file_path>
+
+Flags:
+  -d, --debug     debug mode
+      --force     override the existing trust policies, never prompt
+  -h, --help      help for import
+  -v, --verbose   verbose mode
+```
+
+### notation policy init
+
+```text
+Initialize trust policies with default values
+
+Usage:
+  notation policy init [flags]
+
+Flags:
+  -d, --debug             debug mode
+      --force             override the existing trust polices, never prompt
+  -h, --help              help for export
+      --ts stringArray    specify the trust stores in format "<type>:<name>", e.g. "ca:my_store"
+  -v, --verbose           verbose mode
+```
+
+### notation policy show
+
+```text
+Show trust policies
+
+Usage:
+  notation policy [flags] show
+
+Flags:
+  -d, --debug     debug mode
+  -h, --help      help for show
+  -v, --verbose   verbose mode
+```
+
+## Usage
+
+### Initialize trust policies with default values
+
+```shell
+notation policy init
+```
+
+Upon successful execution, trust policies with default values are created as following. Make sure to update the `trustStores` property with correct values.
+
+```jsonc
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "policy-by-init-command",
+            "registryScopes": ["*"],
+            "signatureVerification": {
+                "level": "strict"
+            },
+            "trustStores": ["ca:default"],
+            "trustedIdentities": ["*"]
+        }
+    ]
+}
+
+```
+
+Use `--force` flag to override existing policies without prompt.
+
+### Initialize trust policies with specified trust stores
+
+```shell
+notation policy init --ts "ca:my_store" --ts "ca:my_store_2"
+```
+
+Upon successful execution, trust policies with default values and specified trust stores are created as following:
```

Comment:
Again, we should print out the generated trust policy statement as the command's output.